web/src/main/java/no/nav/modiapersonoversikt/service/pdl/PdlOppslagService.kt

```diff
@@ -37,7 +37,8 @@ interface PdlOppslagService {
     data class PdlKriterie(
         val felt: PdlFelt,
         val value: String?,
-        val boost: Float? = null
+        val boost: Float? = null,
+        val searchHistorical: Boolean? = false
```

Comment:
For vår egen forstårelse i fremtiden kan vi vurdere å lage en enum-verdi for `searchHistorical`: 
```suggestion
        val searchHistorical: PdlSokeOmfang = PdlSokeOmfang.HISTORISK_OG_GJELDENDE
```

Så må man evt bare lage en enum med tre verdier.